packages/labs/analyzer/src/lib/model.ts

```diff
@@ -585,11 +602,42 @@ export class CustomElementDeclaration extends ClassDeclaration {
   constructor(init: CustomElementDeclarationInit) {
     super(init);
     this.tagname = init.tagname;
+    this.attributes = init.attributes;
     this.events = init.events;
     this.slots = init.slots;
     this.cssProperties = init.cssProperties;
     this.cssParts = init.cssParts;
   }
+
+  #addFieldToAttributeMap(
+    field: ClassField,
+    attributes: Map<string, Attribute>
+  ) {
+    if (field instanceof CustomElementField && field.attribute) {
+      const {attribute, name: fieldName, ...rest} = field;
+      const name =
+        typeof attribute === 'boolean' ? fieldName.toLowerCase() : attribute;
```

Comment:
`typeof attribute === 'boolean'` should be the same as `attribute === true`, right? I find the later a tad easier to read.

Reply:
i think this is a leftover from an earlier iteration. at this point, the `attribute` on the field model should always be a `string | undefined`, never a bool